cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -171,6 +171,10 @@ public BuildCommand() {
             " the services in the current package")
     private Boolean exportOpenAPI;
 
+    @CommandLine.Option(names = "--export-component-model", description = "generate component model for" +
+            " the services in the current package", hidden = true)
```

Comment:
Shall we improve the command option help text to something similar to the the below? WDYT?
```suggestion
    @CommandLine.Option(names = "--export-component-model", description = "generate a model to represent interactions in between the package components (i.e. service/type definitions) and, export in JSON format", hidden = true)
```